### DISCUSSION TITLE
启动sh start_server.sh时出错

### DISCUSSION THREAD

**Discussion by jj03749789** [kbengine/kbengine#1587] [Q&A]
cellapp-1: [ERROR]: Components::findComponents: receive error!
  loginapp-1: [ERROR]: BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
  loginapp-1: [ERROR]: Components::findComponents: receive error!
   baseapp-1: [ERROR]: BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
   baseapp-1: [ERROR]: Components::findComponents: receive error!
cellappmgr-1: [ERROR]: BundleBroadcast::receive: failed! It can be caused by the firewall, the broadcastaddr, etc.Maybe broadcastaddr is not a LAN ADDR, or the Machine process is not running.
cellappmgr-1: [ERROR]: Components::findComponents: receive error!

这个问题怎么解决呢？运行环境是Linux centos7

**Comment by Wandar** [comment]


**Reply by jj03749789** [reply]
关掉防火墙后可以了，感谢
